<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saldo</class>
 <widget class="QDialog" name="saldo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> *
{
font: century gothic;
color:white;
}
QPushButton
{
background:#e9de72;
border-radius:60px;

}
QPushButton
{
color:;
border-radius:15px;
}
QFrame
{
background:#b5aa47;
border-radius:15px;
}
QTextEdit
{
font: century gothic;
color:white;
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelOtsikkoKortti">
      <property name="font">
       <font>
        <family>gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Pankkikortti</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelKorttiNumero">
      <property name="font">
       <font>
        <family>gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Numero: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelKorttityyppi">
      <property name="font">
       <font>
        <family>gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="labelOtsikkoSaldo">
      <property name="font">
       <font>
        <family>gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Tililt√§ nostettavissa oleva saldo:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelNostettavaSaldo">
      <property name="font">
       <font>
        <family>gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelOtsikkoTili">
      <property name="font">
       <font>
        <family>gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Pankkitili</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTilinumero">
      <property name="font">
       <font>
        <family>gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Numero: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTiliOmistaja">
      <property name="font">
       <font>
        <family>gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Omistaja: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelPvm">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>155</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>gothic</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Pvm: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelKlo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>205</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>gothic</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Klo: </string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="labelSaldo">
      <property name="font">
       <font>
        <family>gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Saldo: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelLuottoraja">
      <property name="font">
       <font>
        <family>gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Luottoraja: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>811</width>
     <height>571</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnTakaisin">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>gothic</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Takaisin</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOtsikko">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>gothic</family>
      <pointsize>26</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>SALDO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget_3</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>labelPvm</zorder>
  <zorder>labelKlo</zorder>
  <zorder>horizontalLayoutWidget_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
